chapter_convolutional-neural-networks/pooling.md

```diff
@@ -284,19 +256,20 @@ Below, we will concatenate arrays `X` and `X+1`
 on the channel dimension to construct an input with 2 channels.
 
 ```{.python .input}
-X = np.concatenate((X, X + 1), axis=1)
+#@tab mxnet, pytorch
+X = d2l.concat((X, X + 1), 1)
 X
 ```
 
 ```{.python .input}
-#@tab pytorch
+
 X = torch.cat((X, X + 1), dim=1)
 X
 ```
 
 ```{.python .input}
 #@tab tensorflow
-X = tf.reshape(tf.stack([X, X+1], axis=0), (1, 2, 4, 4))
+X = tf.reshape(tf.stack([X, X+1], 0), (1, 2, 4, 4))
```

Comment:
This can be d2l.reshape instead?

Reply:
quite different to the preview cell, so not necessary to use d2l here.

Reply:
I thought the idea is to now use `d2l` as much as possible, even if the code blocks are not intended to be merged.
Anyways they will be changed back using reverse aliases, so why not use them everywhere possible for more consistency.